src/main/resources/jsonSchemas/antoraPlaybookSchema.json

```diff
@@ -362,6 +362,14 @@
           ],
           "description": "Tags as comma-separated list, an array or a list. Negate entries by prepending with a '!', use '*' as a wildcard."
         },
+        "version": {
+          "type": [
+            "string",
+            "boolean",
+            "object"
+          ],
+          "description": "Fallback version(s). Either a fixed string value, 'true' to use the refname, or an object of patterns (the keys) and replacements (the values)."
+        },
```

Comment:
Should we include examples as well, or a link to the documentation?

Reply:
Again - not sure if links are support in the IDE. Please give it a try; if they are, feel free to add a link.

Reply:
IntelliJ supports a secondary attribute "x-intellij-html-description" with a HTML descriptor of a file which then also supports links. I updated the PR to include this. If you want to add such a descriptor for other fields as well please create a new issue and PR.